src/nrnoc/syn.mod

```diff
@@ -3,7 +3,7 @@ a synaptic current with alpha function conductance defined by
         i = g * (v - e)      i(nanoamps), g(microsiemens);
         where
          g = 0 for t < onset and
-         g = gmax * (t - onset)/tau * exp(-(t - onset - tau)/tau)
```

Comment:
The ```- tau``` looks strange but is correct given that when ``` t -  onset == tau``` the value is gam since exp(0) == 1.0